source/agora/consensus/EnrollmentManager.d

```diff
@@ -190,6 +190,34 @@ public class EnrollmentManager
         return this.validator_set.getEnrolledHeight(enroll_hash);
     }
 
+    /***************************************************************************
+
+        Add a enrollment data to the enrollment pool
+
+        Params:
+            enroll = the enrollment data to add
+            block_height = current block height
+            finder = the delegate to find UTXOs with
+
+        Returns:
+            true if the enrollment data has been added to the enrollment pool
+
+    ***************************************************************************/
+
+    public bool addEnrollment (const ref Enrollment enroll, Height block_height,
+        scope UTXOFinder finder) @safe nothrow
+    {
+        auto enrolled_height = this.getEnrolledHeight(enroll.utxo_key);
+        ulong avail_height;
```

Comment:
Annotation for avail_height is required.

Reply:
> Annotation for avail_height is required.

It would be better to describe the purpose of the `avail_height`. Thanks.

Reply:
> Annotation for avail_height is required.

It would be better to describe the purpose of the avail_height. Thanks.